hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/SCMContainerManager.java

```diff
@@ -282,6 +290,12 @@ public ContainerInfo allocateContainer(final ReplicationType type,
   public void deleteContainer(ContainerID containerID) throws IOException {
     lock.lock();
     try {
+      Set<ContainerReplica> replicas =
+          containerStateManager.getContainerReplicas(containerID);
+      for(ContainerReplica replica : replicas){
+        nodeManager.removeContainers(replica.getDatanodeDetails(), Stream.of(
```

Comment:
Can we create the set of containerID outside the loop with a simpler Java APIs:

Set<ContainerID> containerIdSet = Collections.singleton(containerID);
replias.forEach(replica->replica.removeContainers(replica.getDatanodeDetails(), containerIdSet));